LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -253,11 +265,16 @@ size_t formatProcessLog(char* chBuf)
     index = bufferCopy(chBuf, chBufCpy, index);
 
     // truncate, in the unlikely event of a long logline > |BUFSIZE-85|
-    // leave at least 36 slots to close the JSON with `..."},\"SchemaVersion\":\"1.0.0\"}\n`
+    // leave at least 36 slots to close the JSON/XML

```

Comment:
can qualify this comment to say: 36 slots for JSON or 21 slots for XML.

Reply:
Commenting change made